app_dart/test/request_handlers/push_gold_status_to_github_test.dart

```diff
@@ -1312,16 +1417,16 @@ String tryjobEmpty() {
 }
 
 /// JSON response template for Skia Gold empty tryjob status request.
```

Comment:
Nit while you're here
```suggestion
/// JSON response template for Skia Gold untriaged tryjob status request.
```

Looks like an old copy paste typo